packages/realm-react/README.md

```diff
@@ -46,69 +44,170 @@ class Task extends Realm.Object {
     return {
       _id: new Realm.BSON.ObjectId(),
       description,
-      isComplete: false,
-      createdAt: new Date()
+      createdAt: new Date(),
     };
   }
 
-  // To use a class as a Realm object type, define the object schema on the static property "schema".
   static schema = {
     name: 'Task',
     primaryKey: '_id',
     properties: {
       _id: 'objectId',
       description: 'string',
-      isComplete: {type: 'bool', default: false},
+      isComplete: { type: 'bool', default: false },
       createdAt: 'date'
     },
   };
 }
 
-export default createRealmContext({schema: [Task]});
+const { RealmProvider, useRealm, useQuery } = createRealmContext({ schema: [Task] })
+
+export default function AppWrapper() {
+  return (
+    <RealmProvider><TaskApp /></RealmProvider>
+  )
+}
+
+function TaskApp() {
+  const realm = useRealm();
+  const tasks = useQuery(Task);
+  const [newDescription, setNewDescription] = useState("")
+
+  return (
+    <SafeAreaView>
+      <View style={{ flexDirection: 'row', justifyContent: 'center', margin: 10 }}>
+        <TextInput
+          value={newDescription}
+          placeholder="Enter new task description"
+          onChangeText={setNewDescription}
+        />
+        <Pressable
+          onPress={() => {
+            realm.write(() => {
+              realm.create("Task", Task.generate(newDescription));
+            });
+            setNewDescription("")
+          }}><Text>➕</Text></Pressable>
+      </View>
+      <FlatList data={tasks.sorted("createdAt")} keyExtractor={(item) => item._id.toHexString()} renderItem={({ item }) => {
+        return (
+          <View style={{ flexDirection: 'row', justifyContent: 'center', margin: 10 }}>
+            <Pressable
+              onPress={() =>
+                realm.write(() => {
+                  item.isComplete = !item.isComplete
+                })
+              }><Text>{item.isComplete ? "✅" : "☑️"}</Text></Pressable>
+            <Text style={{ paddingHorizontal: 10 }} >{item.description}</Text>
+            <Pressable
+              onPress={() => {
+                realm.write(() => {
+                  realm.delete(item)
+                })
+              }} ><Text>{"🗑️"}</Text></Pressable>
+          </View>
+        );
+      }} ></FlatList>
+    </SafeAreaView >
+  );
+}
 ```
 
-Wrap the component needing access to Realm (possibly your entire application) with the `RealmProvider` componenet.
-The `RealmProvider` also accepts Realm configuration properties.
+For a full fledged example, check out [our templates](https://github.com/realm/realm-js#template-apps-using-expo-for-react-native).
+
+## Realm Hooks
+### useQuery
+
+Returns a [Realm.Result] from a given type. This Hook will update on any changes to any object in the collection and return an empty array if the collection is empty.
```

Comment:
```suggestion
Returns a [Realm.Result] from a given type. This Hook will update on any changes to any Object in the Collection and return an empty array if the Collection is empty.
```

Reply:
The React Native types are capitalized, so I think the Realm ones should be as well.

Reply:
ooo...and I just realized I wanted to make links to our API docs.  I'll change that as well.